fashrevwall/TwitterClient.py

```diff
@@ -0,0 +1,62 @@
+"""
+Represents a client that connects to Twitter to retrieve tweets based on a
+search criteria using Twitter REST API and Tweepy.
+"""
+import json
+import tweepy
+from tweepy import OAuthHandler
+from TwitterStreamListener import TwitterStreamListener
```

Comment:
absolute imports would be better `from .TwitterStreamListener import TwitterStreamListener`